ui/components/MeshModelRegistry/MeshModelComponent.js

```diff
@@ -0,0 +1,518 @@
+import { Button, withStyles } from '@material-ui/core';
+import { withSnackbar } from 'notistack';
+import React, { useState, useEffect } from 'react';
+import { Paper } from '@material-ui/core';
```

Comment:
We can move forward with this PR however, this is something to note if we import only a few components from mui like this then let's not use named imports, The reason behind this is that when we import using named imports, it not only brings in the specific component (in this case, the Paper component) but the entire MUI core library, leading to an increase in the package size.

Good thing is Nextjs v13 has a solution for this where we can use named import without increasing package size however as we are not using NextJs v13 right now, let's avoid using named imports.